src/libcollections/borrow.rs

```diff
@@ -171,6 +203,18 @@ impl<'a, B: ?Sized> Cow<'a, B> where B: ToOwned {
     /// Acquire a mutable reference to the owned form of the data.
     ///
     /// Copies the data if it is not already owned.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::borrow::Cow;
+    ///
+    /// let mut cow: Cow<&str> = Cow::Owned("Hello");
```

Comment:
I think that this example may want to be redone a bit, the `Owned` variant is meant to represent something like `String`, and you probably don't want to call `to_mut` on `Cow<str>` because that will give you the type `&mut str` which isn't super useful.

Perhaps this could be `Cow<[_]>` where the owned variant is `vec![1, 2, 3]` and then you call `.to_mut()` to modify one of the elements and then assert that the `cow` is equal to a slice?